src/docs/GetStarted/Developers/AdeoPreset/index.mdx

```diff
@@ -17,7 +17,7 @@ Here's how to use this feature:
 
 The file `mozaic.config.js` allows you to extend the configuration of Mozaic _(you can learn more about this file by [reading the dedicated page](/GetStarted/Developers/Configuration/))_.
 
-In order to use the **Adeo preset**, you must create at the root of your project a file `mozaic.config.js` _(if it does not already exist)_ and follow these steps:
+To use the **Adeo preset**, you must create at the root of your project a file `mozaic.config.js` _(if it does not already exist)_ and follow these steps:
```

Comment:
You must create a file `mozaic.config.js` at the root of your project to use the **Adeo preset** _(if it does not already exist)_ and follow these steps: